roles/serviceassurance/tasks/component_alertmanager.yml

```diff
@@ -0,0 +1,56 @@
+- name: Set default Alertmanager manifest
+  set_fact:
+    alertmanager_manifest: |
+      apiVersion: monitoring.coreos.com/v1
+      kind: Alertmanager
+      metadata:
+        labels:
+          alertmanager: '{{ meta.name }}'
+        name: '{{ meta.name }}'
+        namespace: '{{ meta.namespace }}'
+      spec:
+        replicas: 1
+        serviceAccountName: prometheus-k8s
+        serviceMonitorSelector:
+          matchLabels:
+            app: smart-gateway
+        storage:
+          class: {{ alertmanager_storage_class }}
+          resources: {{ alertmanager_storage_resources }}
+          selector: {{ alertmanager_storage_selector }}
+          volumeClaimTemplate:
+            spec:
+              resources:
+                requests:
+                  storage: {{ alertmanager_pvc_storage_request }}
+  when: alertmanager_manifest is not defined
+
+- name: Set default Alertmanager configuration manifest
+  set_fact:
+    alertmanager_config_manifest: |
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: alertmanager-'{{ meta.name }}'
+        namespace: '{{ meta.namespace }}'
+      type: Opaque
+      stringData:
+        alertmanager.yaml: |-
+          global:
+            resolve_timeout: 5m
+          route:
+            group_by: ['job']
+            group_wait: 30s
+            group_interval: 5m
+            repeat_interval: 12h
+            receiver: 'null'
+          receivers:
+          - name: 'null'
+  when: alertmanager_config_manifest is not defined
+
+- name: Create an instance of Alertmanager
+  k8s:
+    definition:
+      '{{ alertmanager_config_manifest }}'
```

Comment:
You need to create the Alertmanager instance as well via `alertmanager_manifest`